pkg/accountGRPC/controller/controller.go

```diff
@@ -0,0 +1,174 @@
+package accountgrpccontroller
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/google/uuid"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+	"google.golang.org/protobuf/types/known/emptypb"
+
+	customerrors "github.com/stasBigunenko/monorepa/errors"
+	"github.com/stasBigunenko/monorepa/model"
+	pb "github.com/stasBigunenko/monorepa/pkg/accountGRPC/proto"
+)
+
+type AccountGRPCСontroller struct {
+	client pb.AccountGRPCServiceClient
+}
+
+func New(cli pb.AccountGRPCServiceClient) AccountGRPCСontroller {
+	return AccountGRPCСontroller{
+		client: cli,
+	}
+}
+
+func (s AccountGRPCСontroller) formatError(err error, message string) error {
+	st, ok := status.FromError(err)
+	if !ok {
+		return fmt.Errorf("%s, failed to parse status or not a grpc error type: %w", message, err)
+	}
+
+	if st.Code() == codes.NotFound {
+		return fmt.Errorf("%s: %w", message, customerrors.NotFound)
+	}
+
+	if st.Code() == codes.AlreadyExists {
+		return fmt.Errorf("%s: %w", message, customerrors.AlreadyExists)
+	}
+
+	if st.Code() == codes.DeadlineExceeded {
+		return fmt.Errorf("%s: %w", message, customerrors.DeadlineExceeded)
+	}
+
+	return fmt.Errorf("%s: %s", message, err.Error())
+}
+
+func (s AccountGRPCСontroller) CreateAccount(userID uuid.UUID) (uuid.UUID, error) {
```

Comment:
It would be convenient if `CreateAccount` accept ctx as parameter. 

Than you can have context chain from the http handler to storage.

Reply:
We will add it in a separate task.